From 5738b7046839f48e016db9ac4abad6e8a49d42ee Mon Sep 17 00:00:00 2001
From: Jon Nordby <jononor@gmail.com>
Date: Fri, 25 Apr 2014 17:14:58 +0200
Subject: Check for and run intltoolize in autogen.sh

On some platforms, including Heroku, the intltool macros cannot
be found during build, possibly because the version of autoreconf
does not respect ACLOCAL_PATH. Using intltoolize also allows
to avoid a dependency on intltool when installing from tarball.

diff --git a/autogen.sh b/autogen.sh
index aa5b7a9..0f34263 100755
--- autogen.sh
+++ autogen.sh
@@ -21,6 +21,13 @@ if test -z $AUTORECONF; then
         exit 1
 fi
 
+INTLTOOLIZE=`which intltoolize`
+if test -z $INTLTOOLIZE; then
+        echo "*** No intltoolize found, please install the intltool package ***"
+        exit 1
+fi
+
+intltoolize --automake --copy --force
 autoreconf --force --install --verbose
 
 cd $ORIGDIR

--- configure.ac.orig	2015-04-10 09:49:54.000000000 -0700
+++ configure.ac	2015-04-10 09:52:54.000000000 -0700
@@ -485,8 +485,7 @@
 ]))
 LIBS=$gegl_save_LIBS
 
-m4_ifdef([IT_PROG_INTLTOOL],[IT_PROG_INTLTOOL(0.40.1)],
-  AC_MSG_ERROR([*** intltool version 0.40.1 or later required]))
+IT_PROG_INTLTOOL(0.40.1)
 
 AM_GLIB_GNU_GETTEXT
 
